middlewares/userValidations.js

```diff
@@ -0,0 +1,100 @@
+import models from '../models';
+
+const { user } = models;
```

Comment:
Find a way to ensure that the models always start with uppercase so that this won't be confusing. In essence, `User` instead of `user`

Reply:
ok. will do that